druid/examples/hello.rs

```diff
@@ -12,62 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! This is a very small example of how to setup a druid application.
+//! It does the almost bare minimum while still being usefull.
+
+use druid::widget::prelude::*;
```

Comment:
do we actually need the widget prelude? I normally only import this if I am implementing the `Widget` trait.

Reply:
I thought the idiomatic thing was to just always import the prelude if you need basically anything from it.
I think just always having it is a good way to avoid editors adding individual imports instead of just this prelude.